styles/main.css

```diff
@@ -267,25 +268,25 @@ div.favourites input:checked + label {
 div.gallery {
     display: grid;
     grid-template-columns: 1fr 1fr 1fr 1fr;
+    gap: 10px;
     
 }
 
 div.gallery img {
     display: block;
     width: 100%;
     height: 100%;
-    padding: 5px;
     object-fit: cover;
```

Comment:
You create grid gutters using the `gap` property, not by adding padding to the grid items